collect_stata/stata_to_json.py

```diff
@@ -1,18 +1,18 @@
-"""stata_to_json.py"""
+"""Read stata files and write out json files.
+"""
 __author__ = "Marius Pahl"
 
 from .dataset import Dataset
 
 
-def stata_to_json(study_name, input_path, output_path):
-    """
-    Input:
-    study_name: Name of the study
-    input_path: path to data folder
-    output_path: path to output folder
+def stata_to_json(study_name: str, input_path: str, output_path: str):
+    """Method that reads all stata files from input path and
+    writes out json files
```

Comment:
Please keep to the [pep 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)
"Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. "